deploy/handler/operator.yaml

```diff
@@ -279,6 +281,28 @@ spec:
             initialDelaySeconds: 5
             periodSeconds: 5
             timeoutSeconds: 1
+        - args:
+          - --logtostderr
+          - --secure-listen-address=:8443
+          - --upstream=http://127.0.0.1:8080
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+              - ALL
+          image: quay.io/openshift/origin-kube-rbac-proxy:4.10.0
+          imagePullPolicy: IfNotPresent
+          name: kube-rbac-proxy
+          ports:
+          - containerPort: 8443
+            name: metrics
+            protocol: TCP
+          resources:
+            requests:
+              cpu: 10m
+              memory: 20Mi
+          terminationMessagePolicy: FallbackToLogsOnError
+
```

Comment:
why is this needed in this PR?

Reply:
You need a kube-rbac-proxy to allow access to the metrics from prometheus